src/compile.ts

```diff
@@ -309,7 +309,10 @@ export enum ${def.name.value} {
   `
   }
 
-  write(Preamble)
+  let preamble = await readPreamble();
+  if(!preamble) process.exit(0);
```

Comment:
It seems to me like an error would be more appropriate here. 

p.s: I'm considering making `compile.ts` usable as a library to make it easier to integrate with graphql-code-generator - so throwing an ordinary error here will work best (I'll probably switch it to a custom error type later)